inheritance/src/main/java/inheritance/Restaurant.java

```diff
@@ -0,0 +1,61 @@
+package inheritance;
+
+import java.util.ArrayList;
+
+public class Restaurant {
+    private String name;
+    private double numberOfStars;
+    private String priceCategory;
+
+    private ArrayList<Review> reviews;
+    private int cummulReviews;
+    private double cummulStars;
+    private double avgRating;
+
+    public Restaurant(String name, double avgRating, String priceCategory) {
+        this.name = name;
+        this.numberOfStars = numberOfStars;
+        this.priceCategory = priceCategory;
+        this.reviews = new ArrayList<>();
+        this.cummulReviews = 0;
+        this.cummulStars = 0;
+        this.avgRating = 0;
+    }
+
+    public String getName() {
+         return name;
+    }
+
+    public int getCummulReviews() {
+        return cummulReviews;
+    }
+
+    public double getCummulStars() {
+        return cummulStars;
+    }
+
+    public double getNumberOfStars() {
+        return numberOfStars;
+    }
+
+    public String getPriceCategory() {
+        return priceCategory;
+    }
+
+    public void addReview(Review review){
+        if(review.getRestaurant() == this.getName()) {
```

Comment:
Nice! You definitely don't want to allow someone to add a review for another restaurant. What about preventing users from adding the same review more than once?